tests/test_backend.py

```diff
@@ -742,21 +743,26 @@ def test_write_dataset_compression(h5file):
     nchunks = int(np.ceil(data.size / chunksize))
 
     slices1 = write_dataset(
-        h5file, "test_data", data, compression="gzip", compression_opts=3
+        h5file,
+        "test_data",
+        data,
+        filters=Filters(compression="gzip", compression_opts=3),
```

Comment:
This API is definitely less ergonomic than before. However, these are internal functions and this bit of ugliness is limited to a handful of unit tests.